webview/src/experiments/components/table/body/RowContextMenu.tsx

```diff
@@ -269,6 +269,12 @@ const getSingleSelectMenuOptions = (
       'Create new Branch',
       MessageFromWebviewType.CREATE_BRANCH_FROM_EXPERIMENT
     ),
+    disableIfRunning(
+      'Rename Experiment',
+      MessageFromWebviewType.RENAME_EXPERIMENT,
```

Comment:
Renaming an experiment clears the experiment of its starred or plotted state:

https://github.com/iterative/vscode-dvc/assets/43496356/af9878b5-bc95-40b6-a2f4-a07851d7bdd3

Since I don't think there is an easy way to fix this,  I think you can keep it like it is for now and we can look into fixing it at a later date.

Reply:
> Renaming an experiment clears the experiment of its starred or plotted state:
> 
> https://github.com/iterative/vscode-dvc/assets/43496356/af9878b5-bc95-40b6-a2f4-a07851d7bdd3
> 
> Since I don't think there is an easy way to fix this,  I think you can keep it like it is for now and we can look into fixing it at a later date. 

Oh wow that's interesting, I didn't even think of something like that happening. Should I create an issue in the backlog just to have this noted somewhere?

Reply:
> Should I create an issue in the backlog just to have this noted somewhere?

Yes, sounds good!